src/main/resources/webroot/monitoring.html

```diff
@@ -213,6 +213,6 @@
     {{/each}}
 </script>
 
-<script src="custom-js/monitoring.js"></script>
+<script src="custom-js/monitoring.js?v=1.0.2"></script>
```

Comment:
@kevinsafon I think this was specific to TransitScreen's internal modifications?  If so, could you please remove from this PR?